File: SEP3_UserSystem/src/main/java/information/InformationPackage.java
package information;

public class InformationPackage {
    private InformationType informationType;

    public InformationPackage(InformationType informationType)
    {
        this.informationType = informationType;
    }

    public InformationType getInformationType()
    {
        return informationType;
    }

    @Override public String toString()
    {
        return "[" + informationType + "]";
    }
}

File: SEP3_ChatSystem/Mediator/Information/InformationPackage.cs
﻿namespace SEP3_ChatSystem.Mediator.Information
{
    public class InformationPackage
    {
        public InformationType informationType { get; set; }
        public string Keyword { get; set; }

        public InformationPackage(InformationType informationType,string keyword)
        {
            this.informationType = informationType;
            Keyword = keyword;
        }

        public InformationPackage() {}

        public InformationType GetInformationType()
        {
            return informationType;
        }

        public string GetKeyword() {
            return Keyword;
        }

        public override string ToString()
        {
            return "[" + informationType + "]";
        }
    }
}